src/uct/sm/scopy/base/scopy_iface.c

```diff
@@ -16,6 +16,10 @@
 #include <uct/sm/base/sm_iface.h>
 
 
+#define UCT_SCOPY_IFACE_DEFAULT_OVERHEAD 2e-6
+#define UCT_SCOPY_IFACE_OVERHEAD         500e-9
```

Comment:
can you pls add comment explaining why we need 2 values

Reply:
added